test/scripts/goal_subcommand_sanity.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+echo "goal subcommand sanity check"
+set -e
+set -x
+
+BINDIR=$0
+TEMPDIR=$1
```

Comment:
ahh..
```
BINDIR=$1
TEMPDIR=$2
```

Reply:
sorry, got lazy and was letting travis run the test for me